crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -55,6 +55,7 @@ use super::{
 };
 use crate::events::SyncTimelineEventWithoutContent;
 
+#[derive(Clone)]
```

Comment:
Still needed?

Reply:
I am using it here: https://github.com/matrix-org/matrix-rust-sdk/pull/2118/files#diff-4638fca028fc6f81c6c740b6c503361149c9c96dfd4589068c9df7fe98a17e3fR470

Reply:
Okay, let's keep it for now.